Source/ARTAuth+Private.h

```diff
@@ -2,8 +2,6 @@
 //  ARTAuth+Private.h
```

Comment:
These are superfluous. They also add maintenance overhead (if this file is renamed) and cognitive load for maintainers, inspectors and reviewers alike.

Reply:
here db1308345753fbe2e41eea5d247233ad3afd2934

Reply:
@QuintinWillison I can't merge it without resolving this conversation - please resolve it (if it's ok) or add another comment (if it's not). Thanks

Reply:
Sorry for the delay in responding to this. I'm resolving this conversation now but will leave it for you to land the PR.